components/gitpod-cli/cmd/env.go

```diff
@@ -51,24 +52,27 @@ Note that you can delete/unset variables if their repository pattern matches the
 delete environment variables with a repository pattern of */foo, foo/* or */*.
 `,
 	Args: cobra.ArbitraryArgs,
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
 		log.SetOutput(io.Discard)
 		f, err := os.OpenFile(os.TempDir()+"/gp-env.log", os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
 		if err == nil {
 			defer f.Close()
 			log.SetOutput(f)
 		}
 
+		ctx, cancel := context.WithTimeout(cmd.Context(), 1*time.Minute)
+		defer cancel()
+
 		if len(args) > 0 {
```

Comment:
@iQQBot it is not possible to distinguish right now between `gp env` and `gp env foo=bar`

Maybe it is worth another property to the event generally something like `args: number` - which is count of args for the command, wdyt?